src/v2/BaseLoginRouter.js

```diff
@@ -84,6 +85,9 @@ export default Router.extend({
   },
 
   handleIdxResponseSuccess(idxResponse) {
+    // Check if cookie needs to be updated
+    this.updateIdentifierCookie(idxResponse);
```

Comment:
did you check further remediations (next screens) after authentication.. do they contains `dxResponse?.rawIdxState?.success`  ??

Reply:
It shouldn't matter because we're only interested in the time when the user successfully completes authentication, hence only updating the cookie at that time only. From my local tests in my OIE org, I can see it only returns those objects at the time of authentication only.